modules/rosa-sharing-vpc-hosted-zones.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-hosted-zones_{context}"]
+= Updating the shared VPC role and creating hosted zones
+
+After the *cluster-creating AWS account* user provides the DNS domain and the IAM roles, *the VPC-owning AWS account* user must create a private hosted zone and update the trust policy on the IAM role that was created for sharing the VPC.
+
+.Procedure
+
+. The *VPC-owning AWS account* user who owns the VPC must update the VPC sharing IAM role and add the _Installer_ and _Ingress Operator Cloud Credentials_ roles to the principal section of the trust policy.
```

Comment:
To more directly address the user with an imperative statement, consider adding a Prerequisites section and then rewriting this step. For example:
```
.Prerequisites

You must have the VPC-owning AWS account user role that owns the VPC.

.Procedure

. Update the VPC sharing IAM role and add the `Installer` and `Ingress Operator Cloud Credentials` roles to the principal section of the trust policy:
```